qt/scientific_interfaces/ISISReflectometry/test/Runs/RunsPresenterTest.h

```diff
@@ -682,27 +684,31 @@ class RunsPresenterTest : public CxxTest::TestSuite {
     auto algRunner = expectGetAlgorithmRunner();
     presenter.notifyStartMonitor();
     auto expected = defaultLiveMonitorAlgorithmOptions(instrument, updateInterval);
-    assertAlgorithmPropertiesContainOptions(expected, algRunner);
+    assertAlgorithmPropertiesContainOptions(*expected, algRunner);
     verifyAndClear();
   }
 
   void testStartMonitorSetsDefaultPostProcessingProperties() {
     auto presenter = makePresenter();
     auto options = defaultLiveMonitorReductionOptions();
-    expectGetLiveDataOptions(options);
+    expectGetLiveDataOptions(
+        std::make_unique<AlgorithmRuntimeProps>(dynamic_cast<const MantidQt::API::AlgorithmRuntimeProps &>(*options)));
```

Comment:
Since this is intentionally making a copy, we could just simplify this down by calling the original method twice:
`expectGetLiveDataOptions(defaultLiveMonitorReductionOptions())`